### ISSUE TITLE
Corrected Coordinates need to show on map

### ISSUE THREAD

**Issue by cachly** [cachly/bug-tracker#580]
**[Original report](https://bitbucket.org/zedsaidstudio/cachly/issues/580) by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

And ultimately it is a great pity that you only see the original location of a puzzle caches on a map and the list, even if you have corrected the coordinates in the listing of the cache.

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------


```
#!json


UserWaypoints": [{
			"AssociatedAdditionalWaypoint": null,
			"CacheCode": "GC2BPAB",
			"Description": "cats turd333",
			"ID": 12505054,
			"IsCorrectedCoordinate": false,
			"IsUserCompleted": false,
			"Latitude": 50.320816666666673,
			"Longitude": -121.1935166666667,
			"UTCDate": "\/Date(1445929648327-0700)\/",
			"UserID": 6386434
		}]
```

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

Fixed issue #606 and issue #580

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

Fixed issue #606 and issue #580


Former-commit-id: 85b61bcbd61b32ff3cedecd93314862c0b84fc68
